src/function/table/call/current_setting.cpp

```diff
@@ -21,20 +21,20 @@ struct CurrentSettingBindData final : public CallTableFuncBindData {
     }
 };
 
-static void tableFunc(TableFunctionInput& data, DataChunk& outChunk) {
+static common::offset_t tableFunc(TableFuncInput& data, TableFuncOutput& output) {
```

Comment:
From my understanding, the offset_t is the number of tuples that the tableFunc returns.
Should we define a new type for it?